sbws/config.log.default.ini

```diff
@@ -27,20 +27,18 @@ class = handlers.RotatingFileHandler
 formatter = to_file
 args = ('/dev/null', )
 
-## for logging to system log
-#[handler_to_syslog]
-#class=handlers.SysLogHandler
-#formatter=to_syslog
-#level = INFO
-#args = ()
+# for logging to system log
+[handler_to_syslog]
+class=handlers.SysLogHandler
+formatter=to_syslog
+level = INFO
+args = ('/dev/log',)
 
 [formatter_to_stdout]
 
 [formatter_to_file]
 
-## syslog-like formater
-#[formatter_to_syslog]
-## hostname should be added here with a context filter, dunno if there is a way
-## to add it here
-#format = %(asctime)s %(module)s[%(process)s]: <%(levelname)s> %(message)s
-#datefmt = %b %d %H:%M:%S
+# syslog-like formatter
+[formatter_to_syslog]
+format = %(asctime)s %(module)s[%(process)s]: <%(levelname)s> %(message)s
+datefmt = %b %d %H:%M:%S
```

Comment:
See previous commit about letting the user set the format with `to_syslog_format`.

If it isn't necessary to set the date format in order to send messages to the syslog, can we not set it? I've never tried logging to syslog, but I would expect it to set the date format as it wants and not rely on us. But I really don't know.

This section can most likely be empty like the other formatters.

Reply:
> If it isn't necessary to set the date format in order to send messages to the syslog, can we not set it?

it wasn't the case time ago, but i'll check, i think you're right and that's not needed